sdk/communication/azure-communication-phonenumbers/samples/siprouting/delete_sip_trunk_sample_async.py

```diff
@@ -0,0 +1,37 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: delete_sip_trunk_sample_async.py
+DESCRIPTION:
+    This sample shows the usage of asynchronous SIP routing client for deleting the configuration
+    of a single SIP trunk.
+
+USAGE:
+    python delete_sip_trunk_sample_async.py
+    Set the environment variables with your own values before running the sample:
+    1) COMMUNICATION_SAMPLES_CONNECTION_STRING - the connection string in your ACS account
+    2) COMMUNICATION_SAMPLES_TRUNK_FQDN - fqdn of the trunk to be deleted
+"""
+
+import os
+import asyncio
+from azure.communication.phonenumbers.siprouting.aio import SipRoutingClient
+
+class DeleteSipTrunkSampleAsync(object):
+    def __init__(self):
+        connection_string = os.getenv("COMMUNICATION_SAMPLES_CONNECTION_STRING")
+        self._client = SipRoutingClient.from_connection_string(connection_string)
```

Comment:
The clients in samples should be demonstrated as a context manager. The async client in particular will give users warnings if it isn't used in a context manager or closed.

Reply:
Ah - I see you're calling close at the bottom. It would be cleaner to have the sample demonstrate a context manager as this is generally the syntax/pattern we would recommend people use.

Reply:
Added context manager pattern.